dao/cluster_dao.go

```diff
@@ -27,6 +27,7 @@ import (
 type ClusterDao interface {
 	GetAllEntitiesInfoOfNode(nodeName string) []e.EntityInfo
 	GetAllEntitiesInfo() []e.EntityInfo
+	GetAllEntitiesForApp(id string) []e.EntityInfo
```

Comment:
@prakhar2611 Is it supposed to get appId and provide all the entities? In that case we should consider naming the variable appId.

Reply:
Yeah correct, changed the variable name to (appId)